.github/workflows/sync.coredns.yml

```diff
@@ -0,0 +1,54 @@
+name: Sync CoreDNS Content
+on:
+  schedule:
+    - cron:  '4/12 * * * *'
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Checkout CoreDNS
+        uses: actions/checkout@v2
+        with:
+          repository: coredns/coredns
+          path: .coredns
+      -
+        name: Setup Go Env
+        run: |
+          echo "::set-env name=GOPATH::$(go env GOPATH)"
+          echo "::add-path::$(go env GOPATH)/bin"
+        shell: bash
+      -
+        name: Deps
```

Comment:
why do need we need to build coredns?

Reply:
The CoreDNS binary is needed by corecheck afaik.
As we already have the source code building seemed like the easiest solution especially considering we are checking the corefiles within the documentation of that specific commit of coredns anyway

Reply:
hmm, you're right; seems to be heavy weight to just pushing to netifly; we better leave this out and do it off line (for now). Also if coredns doesn't compile we can't deploy (just mentioning this)

Reply:
Deploy on netlify is separate from this.
I'll push a new version, where the corefile check is done in a separate action. That way we have an idea on issues even so we don't cancel a sync etc.